xain_fl/fl/coordinator/coordinator.py

```diff
@@ -247,6 +250,7 @@ def _train_local(
     return theta_update, history, opt_config, metrics
 
 
+# TODO: legacy code, needs refactor
```

Comment:
Please create a ticket for it and link it here.

Reply:
https://xainag.atlassian.net/browse/XP-292

Reply:
I meant in the code, so it will be in the TODO message

Reply:
ah ok, that makes more sense indeed